sympy/printing/str.py

```diff
@@ -74,12 +74,15 @@ def _print_BooleanTrue(self, expr):
     def _print_BooleanFalse(self, expr):
         return "False"
 
+	def _print_Not(self, expr):
+		return '~\(' + expr.args[0] + '\)' 
```

Comment:
@Upabjojr 
Thanks a Lot! I'll do that.
Can you tell me why Travis CI build is failing??
Also why is _print_Not() not getting called for str(Not(x))? (My guess is _print_Basic() is getting called)